src/main/resources/com/nirima/jenkins/plugins/docker/builder/DockerBuilderPublisher/help-pull.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Builds the image with --pull. Pulls the latest version of the base image, instead of using the local one.
```

Comment:
Same as with the noCache help - `<code>`, "If set..." and link to the docker docs.